core/src/test/java/org/apache/iceberg/view/ViewCatalogTests.java

```diff
@@ -63,6 +62,15 @@ public abstract class ViewCatalogTests<C extends ViewCatalog & SupportsNamespace
 
   @TempDir private Path tempDir;
 
+  protected String viewLocation(String... paths) {
+    String location = tempDir.toFile().toURI().toString();
+    location = location.substring(0, location.length() - 1);
```

Comment:
we can't just blindly strip the last character. I think what you want to do here is to strip the last "/". For that you'd want to use `LocationUtil.stripTrailingSlash`:

```
protected String viewLocation(String... paths) {
    StringBuilder location =
        new StringBuilder(LocationUtil.stripTrailingSlash(tempDir.toFile().toURI().toString()));
    for (String path : paths) {
      location.append("/").append(path);
    }

    return location.toString();
  }
```